ui/apps/platform/src/services/DiscoveredClusterService.ts

```diff
@@ -167,11 +167,8 @@ function getValues(arg: SearchFilterValue): string[] | undefined {
     return undefined;
 }
 
-export function replaceSearchFilterName(searchFilter: SearchFilter, name: string | undefined) {
-    return {
-        ...searchFilter,
-        [nameField]: typeof name === 'string' && name.length !== 0 ? [name] : undefined,
-    };
+export function replaceSearchFilterNames(searchFilter: SearchFilter, names: string[] | undefined) {
+    return { ...searchFilter, [nameField]: names };
 }
```

Comment:
Updated implementation to allow multiple name filters, which is supported by the API.